src/Hl7.Fhir.Core/Rest/BaseFhirClient.cs

```diff
@@ -969,7 +969,7 @@ private void setResourceBase(Resource resource, string baseUri)
             var request = tx.Entry[0];
             // tx (-> ITyped)? -> entryRequest 
             // entry -> ITyped -> tx
-            var entryRequest = await request.ToEntryRequestAsync(Settings, _fhirVersion).ConfigureAwait(false);
+            var entryRequest = await request.ToEntryRequestAsync(Settings, _inspector, _fhirVersion).ConfigureAwait(false);
```

Comment:
We have discussed that we want to be able to configure the fhir client with a "pluggable" serializer (so you can use the new ones as well).  I think that once we do that, you will be passing this serializer (or a serializer callback) function down, instead of an inspector.

If this is all internal/private, it doesn't matter for now.

Reply:
yeah, this is not the final solution for now.